### DISCUSSION TITLE
Skopt Progress bar

### DISCUSSION THREAD

**Discussion by antoniocmf** [kernc/backtesting.py#267] [Ideas]
I've started recently running some optimizations using the skopt method and I thought It would be really interesting to add something like a [**tqdm**](https://github.com/tqdm/tqdm) progress bar to the optimization process. 

Unfortunately as I'm new to Python, I wouldn't know how to help implementing this. 

Just something I thought would be nice to share.

Thanks!

**Comment by kernc** [comment]
tqdm is already used for non-skopt optimization. I guess we forgout about it when skopt was introduced.

Please raise this enhancement request on the issue tracker.

**Reply by kernc** [reply]
Filed as https://github.com/kernc/backtesting.py/issues/280.
